server/models/newsSchema.js

```diff
@@ -0,0 +1,19 @@
+var mongoose = require('mongoose');
+var moment = require('moment');
+
+const Schema = mongoose.Schema;
+
+const newsSchema = new Schema(
+    { 
+        event: {type: String, enum: [ 'New Alumni', 'New Student', 'Confirmed Meeting' ] }, 
+        alumni: [{type: Schema.Types.ObjectId, ref: 'Alumni'}], 
+        students: [{type: Schema.Types.ObjectId, ref: 'Student'}], 
+        school: {type: Schema.Types.ObjectId, ref: 'School'},
+        dateCreated: {type: Date, default: moment()}, 
+        role: {type: String, default: 'BOTH', enum: ['BOTH', 'ALUMNI', 'STUDENT']},
+        timeElapsed: {type: String, required: false},
```

Comment:
nit: We don't need to store this, as it will only ever be calculated at display time. We can `newsItem.map (item => Object.assign({}, ...item, timeElapsed: moment(item.dateCreated).fromNow()))` when we're returning a newsItem.

Reply:
It wasn't ever being stored - rather it was a workaround for the fact that each item is a specific document type and would only allow assignment to fields of the model
That said, I've removed the field, and included the map which converts everything to objects before adding timeElapsed